lxd/main.go

```diff
@@ -633,6 +635,10 @@ func setupLXD() error {
 	}
 
 	if *argAuto {
+		if *argStorageBackend == "" {
```

Comment:
Does this mean that passing "lxd init --auto --network-address [::]" will actually cause it to lose its zfs pool if someone had set that up?